apps/api/src/app/workflow-overrides/workflow-overrides.controller.ts

```diff
@@ -43,12 +43,16 @@ import { GetWorkflowOverrideById } from './usecases/get-workflow-override-by-id/
 import { GetWorkflowOverrideByIdCommand } from './usecases/get-workflow-override-by-id/get-workflow-override-by-id.command';
 import { UpdateWorkflowOverrideByIdCommand } from './usecases/update-workflow-override-by-id/update-workflow-override-by-id.command';
 import { UpdateWorkflowOverrideById } from './usecases/update-workflow-override-by-id/update-workflow-override-by-id.usecase';
+import { ApiKeyAuth } from '../shared/framework/swagger/api.key.security';
 
 @ApiCommonResponses()
 @Controller('/workflow-overrides')
 @UseInterceptors(ClassSerializerInterceptor)
 @UseGuards(UserAuthGuard)
 @ApiTags('Workflows-Overrides')
+@ApiBearerAuth()
+@ApiKeyAuth()
+@ApiExcludeController()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** can we maybe deprecate instead and skip during sdk generation?

Reply:
it's two seperate concerns, here is about what we will share in the sdk, exclude is enough